lib/collection/src/shards/shard_holder.rs

```diff
@@ -265,7 +265,14 @@ impl ShardHolder {
             Ok(())
         })?;
 
-        todo!()
+        // TODO(resharding): don't commit resharding here, commit through consensus in driver
+        self.commit_write_hashring(resharding_key.clone())?;
```

Comment:
This is here until we propose hash ring changes through consensus in the driver. That'll be in a separate PR.